### ISSUE TITLE
[Documentation] Offline Install Instructions: MacOS

### ISSUE THREAD

**Issue by G-Ambatte** [naturalcrit/homebrewery#1904]
Document the installation process required to get Homebrewery running on MacOS.

- [ ] Get Homebrewery working on MacOS
- [ ] Document requirements
- [ ] Create install script
- [ ] Create README.MACOS.md

**Comment by calculuschild** [comment]
#1716 was already made by @Gazook89 and as a non-Mac user I only had a couple comments (still waiting on the fixes). Probably a good starting point for this issue if it still needs tweaking.

**Comment by dbolack-ab** [comment]
Did this get "finalized" ?

**Comment by 5e-Cleric** [comment]
closing in favor of discussion at #4678

**Comment by ericscheid** [comment]
I will very soon™ have reason to thoroughly test the MacOS install instructions.

I _might_ have comments.
